File: CSharp/Starter/SpeedOfBuilds/Order.cs
using System;

namespace SpeedOfBuilds
{
    public class Order
    {


        private int quantity;

        private String orderDesc;

        public Order(int quantity, String orderDesc)
        {

            this.quantity = quantity;
            this.orderDesc = orderDesc;
        }

        public override String ToString()
        {
            return "Order [quantity=" + quantity + ", orderDesc=" + orderDesc + "]";
        }

        public int Quantity
        {
            get => quantity;
            set => this.quantity = value;
        }


        public String OrderDesc
        {
            get => orderDesc;
            set => this.orderDesc = value;
        }

    }
}

File: ConcretePractice/SlowTestsCodeFiles/Java/SpeedOfBuilds/src/main/java/com/qa/Order.java
package com.qa;

public class Order {
	
	
	private int quantity;
	
	private String orderDesc;

	public Order(int quantity, String orderDesc) {
		super();
		this.quantity = quantity;
		this.orderDesc = orderDesc;
	}

	@Override
	public String toString() {
		return "Order [quantity=" + quantity + ", orderDesc=" + orderDesc + "]";
	}

	public int getQuantity() {
		return quantity;
	}

	public void setQuantity(int quantity) {
		this.quantity = quantity;
	}

	public String getOrderDesc() {
		return orderDesc;
	}

	public void setOrderDesc(String orderDesc) {
		this.orderDesc = orderDesc;
	}
	
	

}
